testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -1804,6 +1804,28 @@ void testCastToBoolean(CastType castType, SqlOperatorFixture f) {
         consumer);
   }
 
+  @Test void testCodePointsToBytes() {
+    final SqlOperatorFixture f = fixture()
+        .setFor(SqlLibraryOperators.CODE_POINTS_TO_BYTES, VM_FENNEL, VM_JAVA)
+        .withLibrary(SqlLibrary.BIG_QUERY);
+    f.checkFails("^code_points_to_bytes('abc')^",
```

Comment:
Would you like to change the `'abc'` to an array literal?

Reply:
Good suggestion! I add a new test case for string literals array input.